dao/src/main/java/com/fit_goal/util/EventMongoConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.fit_goal.util;
+
+import com.fit_goal.domain.EventDto;
+import lombok.experimental.UtilityClass;
+import org.bson.Document;
+
+import java.time.ZoneId;
+
+import static com.fit_goal.EventDtoFields.*;
+
+
+@UtilityClass
+public class EventMongoConverter {
+
+    public EventDto documentToEventDto(Document document) {
+        EventDto eventDto = new EventDto();
```

Comment:
you can use Lombok's @Builder that will help to create new EventDto easily

Reply:
refactored